cfgov/unprocessed/css/organisms/post-preview.less

```diff
@@ -102,20 +131,49 @@
                 vertical-align: bottom;
             }
         } );
+
+        &_left {
+            .u-webfont-medium();
```

Comment:
This mixin is [deprecated](https://github.com/cfpb/capital-framework/blob/4164814946955b5fb4bee48e02f085219cfcaba5/src/cf-core/src/cf-base.less#L9) and shouldn't be used.